ci/ansible/roles/pulp-test-populate/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Start Pulp
```

Comment:
This role seems to mostly be identical to the `pulp-populate` role. This duplication can be eliminated by making a third role, and then having this and `pulp-populate` role both depend on it. Dependencies can be declared by creating a `{role_name}/meta/main.yml` file:

```yaml
---
dependencies:
  - { role: common, some_parameter: 3 }
  - { role: apache, apache_port: 80 }
  - { role: postgres, dbname: blarg, other_parameter: 12 }
```